target/linux/ramips/dts/mt7621_ampedwireless_ally-00x19k.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_ampedwireless_ally.dtsi"
+
+/ {
+	compatible = "ampedwireless,ally-00x19k", "mediatek,mt7621-soc";
+	model = "Amped Wireless ALLY-00X19K";
+
+};
+
+&switch0 {
+	ports {
+
```

Comment:
drop empty line before and after port@2 here.

Reply:
ok